sources/packages/backend/libs/test-utils/src/factories/supporting-user.ts

```diff
@@ -0,0 +1,46 @@
+import { Application, SupportingUser, SupportingUserType } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+
+/**
+ * Creates a fake supporting user.
+ * @param supportingUserType supporting user type to be created.
+ * @param relations dependencies:
+ * - `application`: application that the supporting user is associated with.
+ * @returns a fake supporting user.
+ */
+export function createFakeSupportingUser(
+  supportingUserType: SupportingUserType,
+  relations: {
+    application: Application;
+  },
+): SupportingUser {
+  const supportingUser = new SupportingUser();
+  supportingUser.application = relations.application;
+  supportingUser.createdAt = new Date();
+  supportingUser.updatedAt = new Date();
+  supportingUser.supportingUserType = supportingUserType;
+  return supportingUser;
+}
+
+/**
+ * Saves a fake supporting user.
+ * @param dataSource data source for the application.
+ * @param supportingUserType supporting user type to be created.
+ * @param relations dependencies:
+ * - `application`: application that the supporting user is associated with.
+ * @returns a persisted fake supporting user.
+ */
+export async function saveFakeSupportingUser(
```

Comment:
The idea of creating the `saveFakeSomething` method was to deal with a situation where there is a proper sequence to save dependent entities to the DB.
Usually, we would not create these types of methods when the only intention is to call the `createFakeSomething` and have it saved.